common/lib/xmodule/xmodule/data.py

```diff
@@ -0,0 +1,25 @@
+"""
+Public data structures for this app.
+
+See OEP-49 for details
+"""
+from enum import Enum
+
+
+class CertificatesDisplayBehaviors(str, Enum):
+    """
+    Options for the certificates_display_behavior field of a course
+
+    end: Certificates are available at the end of the course
+    end_with_date: Certificates are available after the certificate_available_date (post course end)
+    early_no_info: Certificates are available immediately after earning them.
+
+    Only in affect for instructor based courses.
+    """
+    END = "end"
+    END_WITH_DATE = "end_with_date"
+    EARLY_NO_INFO = "early_no_info"
+
+    @classmethod
+    def includes_value(cls, value):
+        return value in set(item.value for item in cls)
```

Comment:
Tiniest of nits: Just want to make sure this was done for readability and not performance. This is perfectly good code as is, but is not any faster than, say, `return any(value==item.value for item in cls)`

Reply:
Yeah, I figured it was more readable while similarly performant. Good callout though.